connectors/sources/confluence.py

```diff
@@ -258,9 +279,18 @@ async def paginated_api_call(self, url_name, **url_kwargs):
                         self.host_url,
                         links.get("next")[1:],
                     )
+            except ServerDisconnectedError:
+                raise
```

Comment:
Can you add a comment (or maybe even a debug log) to help future readers understand why these errors should be re-raised, instead of treated like the exceptions in the below blocks?